<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Document Classifier</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(60, 63, 65);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="start">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <widget class="QPushButton" name="btnStart">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>260</y>
       <width>200</width>
       <height>200</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>wjplay.png</normaloff>wjplay.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="progress">
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>260</y>
       <width>800</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕 Semilight</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="maximum">
      <number>3</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="format">
      <string>%v / 3</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblProgress">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>380</y>
       <width>1000</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕 Semilight</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>PDF -&gt; IMG 변환중</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="coverList">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <widget class="QListWidget" name="lwList">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1280</width>
       <height>620</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕 Semilight</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::NoDragDrop</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::IgnoreAction</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>210</width>
       <height>297</height>
      </size>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="gridSize">
      <size>
       <width>240</width>
       <height>380</height>
      </size>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnToContent">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>630</y>
       <width>200</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕 Semilight</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 163, 139);
color: rgb(255, 255, 255);
border: 0px;</string>
     </property>
     <property name="text">
      <string>다음단계</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnNext">
     <property name="geometry">
      <rect>
       <x>1120</x>
       <y>630</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕 Semilight</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(216, 218, 217);
border: 0px;</string>
     </property>
     <property name="text">
      <string>다음</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCancel">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>630</y>
       <width>200</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕 Semilight</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(139, 163, 255);
color: rgb(255, 255, 255);
border: 0px;</string>
     </property>
     <property name="text">
      <string>선택취소</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPrev">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>630</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕 Semilight</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;
background-color: rgb(216, 218, 217);</string>
     </property>
     <property name="text">
      <string>이전</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbIndex">
     <property name="geometry">
      <rect>
       <x>1050</x>
       <y>690</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕 Semilight</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnComplete">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>630</y>
       <width>200</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕 Semilight</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(200, 120, 140);
color: rgb(255, 255, 255);
border: 0px;</string>
     </property>
     <property name="text">
      <string>완료</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="noCover">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>260</y>
       <width>800</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕 Semilight</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>표지로 분류된 데이터가 없습니다.</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnNoCover">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>580</y>
       <width>240</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕 Semilight</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(139, 163, 255);
color: rgb(255, 255, 255);
border: 0px;</string>
     </property>
     <property name="text">
      <string>완료</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="inputChoice">
    <widget class="QLabel" name="choiceLbl">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>250</y>
       <width>600</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕 Semilight</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>제목별 분류를 수행하시겠습니까?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnNo">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>540</y>
       <width>200</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕 Semilight</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;
background-color: rgb(139, 163, 255);
color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>아니요</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnYes">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>540</y>
       <width>200</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕 Semilight</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;
background-color: rgb(255, 163, 139);
color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>예</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="inputKeyword">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>1200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕 Semilight</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>분류할 제목 문자열(들)을 콤마(,)로 구분하여 입력하여 주십시오.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="leKeyword">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>360</y>
       <width>1000</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕 Semilight</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::LogicalMoveStyle</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btnKeyword">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>520</y>
       <width>200</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕 Semilight</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;
background-color: rgb(255, 163, 139);
color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>입력완료</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDisable">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>520</y>
       <width>200</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕 Semilight</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;
background-color: rgb(140, 140, 140);
color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>입력필요</string>
     </property>
    </widget>
    <zorder>btnDisable</zorder>
    <zorder>label_3</zorder>
    <zorder>leKeyword</zorder>
    <zorder>btnKeyword</zorder>
   </widget>
   <widget class="QWidget" name="ocrProgress">
    <widget class="QLabel" name="lblOCR">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>350</y>
       <width>1000</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>이미지 추출 중</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QProgressBar" name="pbOCR">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>260</y>
       <width>800</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕 Semilight</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="maximum">
      <number>4</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="format">
      <string>%v / 4</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="finish">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>250</y>
       <width>600</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕 Semilight</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>문서 분류가 완료되었습니다.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnFinish">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>540</y>
       <width>200</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕 Semilight</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;
background-color: rgb(255, 163, 139);
color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>종료</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
